hs/Response/Draw.hs

```diff
@@ -22,7 +25,6 @@ drawResponse =
                 )
                 drawScripts
 
-
```

Comment:
I think we do typically have 1 empty line between functions. Can you update this file to be consistent with that rule?